### PR TITLE
SR2023

### PR DESCRIPTION
**Pull request by trickeydan**
All changes for SR2023 should be merged to this branch

### CODE REVIEW

File: sr/robot3/__init__.py [hunk 1]
```diff
@@ -58,11 +58,11 @@
     "INPUT_PULLUP",
     "Note",
     "OUTPUT",
+    "OUT_FIVE_VOLT",
     "OUT_H0",
     "OUT_H1",
     "OUT_L0",
     "OUT_L1",
-    "OUT_L2",
```
Comment (RealOrangeOne) [inline sr/robot3/__init__.py:65]: Question: Where'd this port go?

Comment (raccube) [reply sr/robot3/__init__.py:65]: This one powers the Pi.

Comment (RealOrangeOne) [reply sr/robot3/__init__.py:65]: Not necessarily, right? If a team hooks the Pi up to L1 (which they absolutely can, right?), they may want to use L2 for something else.

Comment (WillB97) [reply sr/robot3/__init__.py:65]: No, only L2 is enabled by default so the brain needs to be connected to this port so it boots and can turn the other ports on.

**Comment by trickeydan** [comment]
I'd like to move this work to `main`, makes more sense than working from a branch

**Review by raccube — APPROVED** [review]

**Review by WillB97 — APPROVED** [review]

**Pull request merged**